resolwe/flow/tests/test_api.py

```diff
@@ -36,14 +36,14 @@ def setUp(self):
         self.data_1 = Data.objects.create(contributor=self.user, slug='test1', process=proc)
         self.data_2 = Data.objects.create(contributor=self.user, slug='test2', process=proc)
 
-    @mock.patch('resolwe.flow.models.Process.objects.all')
-    def test_prefetch(self, process_mock):
+    def test_prefetch(self):
         request = factory.get('/', '', format='json')
         force_authenticate(request, self.user)
-        self.data_viewset(request)
 
         # check that only one request is made to get all processes' types
-        self.assertEqual(process_mock.call_count, 1)
+        # with prefetch 16 queries, without prefetch 17 queries
```

Comment:
I think you should explain in the comment that the number of queries without prefetch depends on the number of Data objects. Just so we know when we add another Data object in the future.